src/Components/Reports/Common/ExecutiveFirstPage.js

```diff
@@ -49,33 +46,37 @@ export const ExecutiveFirstPage = ({ data }) => {
 
     const { stateChartData, stateTableData, conditionsChartData,  conditionsTableData, condtionsInfo } = formattedReportData;
```

Comment:
```suggestion
    const { stateChartData, stateTableData, conditionsChartData,  conditionsTableData, conditionsInfo } = formattedReportData;
```

Reply:
fixed it in the latest commit, good catch @r14chandra!